Tasks/app-store-promote/app-store-promote.ts

```diff
@@ -105,14 +105,18 @@ async function run() {
         tl.debug('Checking for ruby install...');
         tl.which('ruby', true);
 
-        //Whenever a specific version of fastlane is requested, we're going to uninstall all installed
-        //versions of fastlane beforehand.  Note that this doesn't uninstall dependencies of fastlane.
+        //Whenever a specific version of fastlane is requested, we're going to attempt to uninstall any installed
+        //versions of fastlane.  Note that this doesn't uninstall dependencies of fastlane.
         if (installFastlane && fastlaneVersionToInstall) {
-            let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
-            gemRunner.arg(['uninstall', 'fastlane']);
-            tl.debug(`Uninstalling all fastlane versions...`);
-            gemRunner.arg(['-a', '-I']);  //uninstall all versions
-            await gemRunner.exec();
+            try {
+                let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
```

Comment:
Does tl.tool run tl.which for you? I think I remember Luke pointing that out on some PR.

Reply:
I think it does but the lib version being used is 0.9.4 which is really old, so leaving this as  it was being used in the file before.